projects/plugins/jetpack/extensions/store/membership-products/resolvers.js

```diff
@@ -81,11 +87,20 @@ const setDefaultProductIfNeeded = ( selectedProductId, setSelectedProductId, sel
 	}
 };
 
+export const getNewsletterProducts = (
+	productType = PRODUCT_TYPE_PAYMENT_PLAN,
+	selectedProductId = 0,
+	setSelectedProductId = () => {}
+) =>
+	// Returns the products, but silences the snack bar if a default product is created
+	getProducts( productType, selectedProductId, setSelectedProductId, false );
```

Comment:
Optional feedback here. Personally, I find params either to read verses having to look up the fn definition

```suggestion
       $shouldDisplayProductCreationNotice = false;
	getProducts( productType, selectedProductId, setSelectedProductId, shouldDisplayProductCreationNotice );
```

Reply:
I personnally find it too noisy, when one can just look at the definition of `getProducts()` with one click.  But thanks for the suggestion.

Reply:
I personnally find it too noisy, when one can just look at the definition of `getProducts()` with one click.  But thanks for the suggestion.